pkgdown/_pkgdown.yml

```diff
@@ -10,3 +10,12 @@ template:
 
 development:
   mode: auto
+
+articles:
+- title: Working with R6
```

Comment:
Can this just be left as the default `Articles`?

Reply:
Yepp. Removed that.